api/v1beta2/network_types.go

```diff
@@ -461,6 +461,15 @@ type VPCSpec struct {
 	// the API Server.
 	// +optional
 	ElasticIPPool *ElasticIPPool `json:"elasticIpPool,omitempty"`
+
+	// SubnetSchema specifies how CidrBlock should be divided on subnets in the VPC depending on the number of AZs.
+	// PreferPrivate - one private subnet for each AZ plus one other subnet that will be further sub-divided for the public subnets.
+	// PreferPublic - have the reverse logic of PreferPrivate, one public subnet for each AZ plus one other subnet
+	// that will be further sub-divided for the private subnets.
+	// Defaults to PreferPrivate
```

Comment:
It would be worth adding `// +optional` as well.

Reply:
Done